src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/SettingsPluginComponent.kt

```diff
@@ -0,0 +1,50 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.settings
+
+import com.intellij.util.ui.FormBuilder
+import javax.swing.JCheckBox
+import javax.swing.JComponent
+import javax.swing.JPanel
+
+/**
+ * This class displays and captures changes to the values of the Settings entries.
+ */
+class SettingsPluginComponent {
+    var panel: JPanel? = null
+
+    //Checkbox options
+    private var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
+
+    init {
+        createSettingsPanel()
+    }
+
+    /**
+     * Create the main panel for Plugin settings page
+     */
+    fun createSettingsPanel() {
+        panel = FormBuilder.createFormBuilder()
+
+                //Add visual coverage checkbox
+                .addComponent(showCoverageCheckBox, 10)
+                .addComponentFillVertically(JPanel(), 0)
+                .panel
+    }
+
+    /**
+     * Returns the UI component that should be focused when a user opens the TestGenie Settings page.
+     *
+     * @return preferred UI component
+     */
+    fun getPreferredFocusedComponent(): JComponent {
+        return showCoverageCheckBox
+    }
+
+
+    //Settings "changers" (dials, basically)
```

Comment:
What dials does this comment refer to?

Reply:
That was just an intuitive way of thinking about the code below. There are no UI dials present

Reply:
removed the words "(dials, basically)" to avoid future confusion